docs/core.adoc

```diff
@@ -0,0 +1,31 @@
+= AeroGear Services Core SDK
+
+The Core SDK provides helpers for parsing the configuration file and share common utilities between other SDK modules.
+
+It's main features are:
+
+* Parsing of the configuration file
+* Logging
+* Metrics gathering
+
+== Installation
```

Comment:
I wouldn't assume for starters that the SDK is only for Cordova. Actually core itself isn't a plugin, maybe we should say that there is a plugin implementation and point to the package/core-cordova readme? For now there will be 2 packages in NPM: `@aerogearservices/core` and `@aerogearservices/core-cordova`.

Reply:
This is one thing I'm still not 100% clear on, @josemigallas , should the user install both `core` and `core-cordova` in her project? Or is core-cordova going to use core as a dependency to share the non-native js code?

Reply:
Yes that's the goal here, to use only and exclusively `core-cordova`.